privacyidea/static/components/user/views/user.details.html

```diff
@@ -120,6 +120,7 @@ <h3 translate>Tokens for user {{ username }}</h3>
     </div>
     <div class="text-center">
         <button ng-click="enrollToken()" class="btn btn-primary"
+                ng-disabled="checkEnroll()"
```

Comment:
You have an inverted, wrong logic here!

Reply:
fixed in https://github.com/privacyidea/privacyidea/pull/2444/commits/efd10231c5b77d1752c10e224452daef7bf319e8